protobuf/sliverpb/sliver.proto

```diff
@@ -244,8 +245,10 @@ message Download {
   int64 Stop = 5;
   bytes Data = 6;
   bool IsDir = 7;
+  int32 ReadFiles = 8;
+  int32 UnreadableFiles = 9;
 
-  commonpb.Response Response = 9;
+  commonpb.Response Response = 10;
```

Comment:
This needs to remain 9 for backwards compatibility, use 10 for `UnreadableFiles`

Reply:
Ah! Thanks for the info. I did not realize that. Fixed :)